k8s/business-create-ui.kcmap.param

```diff
@@ -5,13 +5,11 @@
 # Action: build
 # Template File: templates/business-create-ui.kcmap.json
 #=========================================================
-NAME="business-create-ui"
+NAME="business-create"
```

Comment:
Why change this? I thought "business-create-ui" was nice and descriptive.

Reply:
Haha I know I wanted it to be this way too, but dev was already setup without the '-ui' (and that includes the way the image was created, ect. which is used by GHA as well) so I had to follow suit